### ISSUE TITLE
unexpected "Downgrading is not supported" message after migration

### ISSUE THREAD

**Issue by Philippe34** [owncloud/core#27016]
### Steps to reproduce
1. Migrate from 9.0.6 to 9.0.7 with ce:9.0.repo
2. "occ upgrade" was a success
3. I tried "occ maintenance" to fix any issues in database, but no errors and changes
4. I restarted apache to clear cache
5. Why does it work for almost owncloud clients, except for 2 clients

### Expected behaviour
No owncloud clients should have 500 Internal Error 

### Actual behaviour
Almost all owncloud clients can connect to owncloud server, except 2 clients.

### Server configuration
**Operating system**: centOS7

**Web server:** apache 2.4.6

**Database:** mariadb 5.5.52

**PHP version:** 5.4.16

**ownCloud version:** 9.0.7

**Updated from an older ownCloud or fresh install:** Updated form 9.0.6 with "yum update"

**Where did you install ownCloud from:** install from ce:9.0.repo

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
Enabled:
  - activity: 2.2.1
  - calendar: 1.4.1
  - comments: 0.2
  - contacts: 1.5.1
  - dav: 0.1.6
  - documents: true
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - mail: true
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
  - user_ldap: 0.8.0
Disabled:
  - encryption
  - external
  - files_antivirus
  - files_external
  - user_external
```

**The content of config/config.php:**
$CONFIG = array (
  'instanceid' => 'ocevydwjehjn',
  'passwordsalt' => 'xxxxxxxxxxxxxxx',
  'secret' => 'xxxxxxxxxxxx',
  'trusted_domains' =>
  array (
    0 => 'cloud.ies.univ-montp2.fr',
  ),
  'datadirectory' => '/owncloud/data',
  'overwrite.cli.url' => 'xxxxxxxx',
  'dbtype' => 'mysql',
  'version' => '9.0.7.1',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'oc_adminown1',
  'dbpassword' => 'xxxxxxxxxxxxxx',
  'installed' => true,
  'forcessl' => true,
  'mail_from_address' => 'cloud',
  'mail_smtpmode' => 'php',
  'mail_domain' => 'ies.univ-montp2.fr',
  'logfile' => '/var/log/owncloud/owncloud.log',
  'loglevel' => 1,
  'ldapIgnoreNamingRules' => false,
  'maintenance' => false,
  'app.mail.server-side-cache.enabled' => true,
  'logtimezone' => 'Europe/Paris',
  'default_language' => 'fr',
  'preview_libreoffice_path' => '/usr/bin/libreoffice',
  'preview_office_cl_parameters' => ' --headless --nologo --nofirststartwizard --invisible --norestore -convert-to pdf -outdir ',
  'theme' => '',
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'trashbin_retention_obligation' => 'auto',
  'updatechecker' => false,
  'ldapUserCleanupInterval' => '240',
  'singleuser' => false,
);
```


Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP

#### LDAP configuration (delete this part if not used)

```
 Configuration                 |                                                                            |
+-------------------------------+----------------------------------------------------------------------------+
| hasMemberOfFilterSupport      | 0                                                                          |
| hasPagedResultSupport         |                                                                            |
| homeFolderNamingRule          |                                                                            |
| lastJpegPhotoLookup           | 0                                                                          |
| ldapAgentName                 |                                                                            |
| ldapAgentPassword             | ***                                                                        |
| ldapAttributesForGroupSearch  |                                                                            |
| ldapAttributesForUserSearch   |                                                                            |
| ldapBackupHost                | xxxxx                                          |
| ldapBackupPort                | 636                                                                        |
| ldapBase                      | ou=xxxxx                                   |
| ldapBaseGroups                | ou=xxxxx                                     |
| ldapBaseUsers                 | ou=xxxxx                                   |
| ldapCacheTTL                  | 600                                                                        |
| ldapConfigurationActive       | 1                                                                          |
| ldapDynamicGroupMemberURL     |                                                                            |
| ldapEmailAttribute            | mail                                                                       |
| ldapExperiencedAdmin          | 1                                                                          |
| ldapExpertUUIDGroupAttr       |                                                                            |
| ldapExpertUUIDUserAttr        |                                                                            |
| ldapExpertUsernameAttr        |                                                                            |
| ldapGroupDisplayName          | cn                                                                         |
| ldapGroupFilter               | objectClass=posixGroup                                                     |
| ldapGroupFilterGroups         |                                                                            |
| ldapGroupFilterMode           | 1                                                                          |
| ldapGroupFilterObjectclass    |                                                                            |
| ldapGroupMemberAssocAttr      | memberUid                                                                  |
| ldapHost                      | ldaps://xxxxxxx                                       |
| ldapIgnoreNamingRules         |                                                                            |
| ldapLoginFilter               | (&(objectclass=*)(uid=%uid))                                               |
| ldapLoginFilterAttributes     |                                                                            |
| ldapLoginFilterEmail          | 0                                                                          |
| ldapLoginFilterMode           | 1                                                                          |
| ldapLoginFilterUsername       | 1                                                                          |
| ldapNestedGroups              | 0                                                                          |
| ldapOverrideMainServer        | 1                                                                          |
| ldapPagingSize                | 1000                                                                       |
| ldapPort                      | 636                                                                        |
| ldapQuotaAttribute            | jOwnCloudQuota                                                             |
| ldapQuotaDefault              |                                                                            |
| ldapTLS                       | 0                                                                          |
| ldapUserDisplayName           | cn                                                                         |
| ldapUserDisplayName2          |                                                                            |
| ldapUserFilter                | (&(objectclass=*)(mail=*)(!(employeeType=past*))(!(jAccountType=generic))) |
| ldapUserFilterGroups          |                                                                            |
| ldapUserFilterMode            | 1                                                                          |
| ldapUserFilterObjectclass     |                                                                            |
| ldapUuidGroupAttribute        | auto                                                                       |
| ldapUuidUserAttribute         | auto                                                                       |
| turnOffCertCheck              | 0                                                                          |
| useMemberOfToDetectMembership | 1


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**
I have no access to ownclient clients

**Operating system:**

#### Web server error log
```
gatsa [24/Jan/2017:03:39:31 +0100] "PROPFIND /owncloud/remote.php/webdav/ HTTP/1.1" 207 382
162.38.134.83 - rosenkrantz [24/Jan/2017:03:39:44 +0100] "GET /owncloud/status.php HTTP/1.1" 500 -
162.38.135.187 - kohler [24/Jan/2017:03:39:45 +0100] "GET /owncloud/status.php HTTP/1.1" 500 -

gatsa : OK
rosenkrantz and kohler : ERROR 500
They were OK before the migration
```

#### ownCloud log (data/owncloud.log)
```
{"reqId":"WIa@8RakaeCUuBD45hRqbQAAAAk","remoteAddr":"162.38.135.187","app":"remote","message":"Downgrading is not supported and is likely to cause unpredictable issues (from 9.0.7.1 to 9.0.6.5)","level":4,"time":"2017-01-24T03:41:53+01:00","method":"GET","url":"\/owncloud\/status.php","user":"48d4c1ec-f7a9-1033-8ccd-cd2c6a1b4911"}
{"reqId":"WIa-EJolYQDIkmnnLdVmAAAAAAM","remoteAddr":"162.38.134.83","app":"remote","message":"Downgrading is not supported and is likely to cause unpredictable issues (from 9.0.7.1 to 9.0.6.5)","level":4,"time":"2017-01-24T03:42:24+01:00","method":"GET","url":"\/owncloud\/status.php","user":"43ba3768-b93f-1032-95cd-030573a4d734"}
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by PVince81** [comment]
Hmm, weird. I know the sync client calls "status.php", but "status.php" should not trigger this message.

Your config.php says 9.0.7.1. Please check the version in the "version.php" file inside the ownCloud folder. It should also have the same version.

It is also possible that ownCloud cached the old version in the sync client's session and didn't clear it for some reason. That could explain why exactly two clients are triggering this error.

If not, then try clearing memcache in case some versions are stored there too.

**Comment by Philippe34** [comment]
Thanks PVince81, it is now resolved.

I checked the "version.php" and it has the save version that config.php.
So I rebooted the server, but there was always the same issue. 
Then I stopped apache and I deleted all session files in /var/lib/php/session.
I started apache and immediatly new session files were created, the error 500 disappeared.

Conclusion: the next time, after the "yum update" and before the "occ upgrade", I will stop apache to close session files access, wait a few minutes to owncloud clients know that service is unavailable and restart apache.
